samples/MqttClient_Hello/app/application.cpp

```diff
@@ -31,17 +26,22 @@ Timer procTimer;
 
 // MQTT client
 // For quick check you can use: http://www.hivemq.com/demos/websocket-client/ (Connection= test.mosquitto.org:8080)
-MqttClient *mqtt;
+#ifdef USE_NEWSTYLECONNECT
```

Comment:
The sample should contain only the recommended way to create an instance of the MqttClient class. Please, remove USE_NEWSTYLECONNECT and use the new recommended constructor.